src/main/scala/no/ndla/articleapi/model/search/SearchableArticle.scala

```diff
@@ -10,18 +10,32 @@ package no.ndla.articleapi.model.search
 
 import java.util.Date
 
-case class LanguageValue[T](lang: Option[String], value: T)
+import no.ndla.articleapi.model.domain.emptySomeToNone
+import no.ndla.articleapi.model.search.LanguageValue.LV
 
-case class SearchableLanguageValues(languageValues: Seq[LanguageValue[String]])
 
-case class SearchableLanguageList(languageValues: Seq[LanguageValue[Seq[String]]])
+object LanguageValue {
```

Comment:
Vi kan vurdere å gjøre om LanguageValue/LV fra en case-class til å være en vanlig klasse, med et companion-object som definerer apply/unapply og med å implementere equals/hashCode selv.

Jeg synes egentlig ikke det ble noe spesielt bedre, men da unngår vi å innføre en ny LV i objektet LanguageValue og kan forholde oss til kun LanguageValue som vi har gjort tidligere.

En implementasjon med vanlig klasse i stedet for case-class kan se slik ut:
```
class LanguageValue[T](val lang: Option[String], val value: T) {
  def canEqual(other: Any): Boolean = other.isInstanceOf[LanguageValue[T]]
  override def equals(other: Any): Boolean = other match {
    case that: LanguageValue[T] =>
      (that canEqual this) && lang == that.lang && value == that.value
    case _ => false
  }

  override def hashCode(): Int = {
    val state = Seq(lang, value)
    state.map(_.hashCode()).foldLeft(0)((a, b) => 31 * a + b)
  }
}
object LanguageValue {
  def apply[T](lang: Option[String], value: T): LanguageValue[T] = new LanguageValue(emptySomeToNone(lang), value)
  def unapply[T](arg: LanguageValue[T]): (Option[String], T) = (arg.lang, arg.value)
}
```

Bestem selv. (Om vi går for en vanlig klasse og companion-object i stedet for en case klasse, så fortjener den sin egen fil om ikke annet 😄